README.md

```diff
@@ -1,46 +1,121 @@
+This repository contains the markdown and static site generator for the https://opencontainers.org website. 
 
 The starter uses the following:
 * **[Hugo](https://gohugo.io/)** as a static site generator
-* **[Bulma](https://bulma.io/)** as a CSS framework
-* **[Netlify](https://www.netlify.com/)** for building, hosting, and DNS management
+* **[Tailwind CSS](https://tailwindcss.com/)** as a CSS framework
+* **[Netlify](https://www.netlify.com/)** for building and hosting.
+
+# Contributing 
+
+To make changes to https://opencontainers.org, create a fork of this repository and submit a pull request. Doing so successfully generates a preview URL using Netlify. 
+
+## Prerequisites 
+
+- Read and abide by the [code of conduct](https://github.com/opencontainers/.github/blob/master/CODE_OF_CONDUCT.md)
+- Sign the [CNCF CLA](https://github.com/kubernetes/community/blob/master/CLA.md)
 
 ## Running locally
 
-Make sure you have [npm](https://www.npmjs.com/) and [yarn](https://yarnpkg.com/) installed. Clone this repository and run the following two commands in its directory:
+Install [npm](https://www.npmjs.com/) and [yarn](https://yarnpkg.com/), then run the following commands in your local repository:
 
 ```shell
-# Install npm assets (just Bulma for Sass/CSS)
 yarn
-
 # Run the server locally
 make serve
 ```
+This installs Hugo and all other dependencies.
+
+## Writing blog posts, news updates, and announcements
+
+The OCI has 3 categories of posts, all stored in the `/content/posts` directory:
+
+- `/news`: news stories from external sources about the OCI
+- `/blog`: blog posts published about by the OCI
+- `/announcements`: press releases published by the OCI or Linux Foundation for distribution
+
+When writing a post:
+
+1. Pick the appropriate category for your post 
+2. Create a markdown file. To keep things easy for others, name it in the following format: `YYYY-MM-DD-your-title-here.md`. 
+3. Use the following frontmatter tags (examples below):
+
+  Tag | Description | Notes |
+  --- | --- | --- |
+  `title` | The post's title. | Required for all posts. Do not include colons (`:`) |
+  `subhead` | The post's [subheadline](https://opencontainers.org/posts/announcements/2018-08-29-oci-welcomes-alibaba-cloud/) – Appears as an H2 above the body text. | Optional for announcements, news. |
+  `author.name` | The author name. Appears below the page title. | Required for all posts. |
+  `author.tag` | The author tag. | Optional. |
+  `tags` | The post's tags. | Optional. |
+  `date` | The post's date, in `YYYY-MM-DD` format. | Required for all posts. | 
+  `readMore` | The external URL of a news article. | Optional for news. (you can include a read more link link in the body too!) |
+
+
+Here's example frontmatter for a **blog** post:
```

Comment:
```suggestion
Here's a sample of front matter for a blog post:
```

Reply:
I get that you're using bold text to differentiate between post types, but I question whether it's helpful. I'm not 💯 on that, so feel free to use bold (sparingly!) at your discretion.